install/operator/boilerplate/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
This is for code contributed to ASF, we should go with something like https://github.com/syndesisio/syndesis/blob/6cd3b2a97b83bf7d9829e407eeec610382c9cc6d/install/operator/config/boilerplate.go.txt